common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -307,33 +307,53 @@ trait CommercialSwitches {
     exposeClientSide = false
   )
 
-  val TrustxBidder: Switch = Switch(
+  val OrielAnalyticsSwitch: Switch = Switch(
     group = Commercial,
-    name = "trustx-bidder",
-    description = "Include TrustX adapter in Prebid auctions",
+    name = "oriel-analytics",
+    description = "Include the analytics script for Oriel to monitor ad-blocking",
     owners = group(Commercial),
     safeState = Off,
     sellByDate = new LocalDate(2018, 2, 21),
+    exposeClientSide = false
+  )
+
+  val prebidSonobi: Switch = Switch(
+    group = Commercial,
```

Comment:
Maybe use a new group for these? I wonder if other prog ad ops might need to change these switches.

Reply:
I think it's going to be safer to build an entirely new switchboard for controlling prebid adaptors, so that the main switchboard is only for devs.  I see this as the first step while it's all under dev control.